package-lock.json

```diff
@@ -1,32 +1,57 @@
 {
   "name": "osc-website",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
   "requires": true,
-  "dependencies": {
-    "@aws-crypto/ie11-detection": {
+  "packages": {
+    "": {
+      "name": "osc-website",
+      "version": "0.0.0",
+      "license": "GPL-3.0-or-later",
+      "dependencies": {
+        "body-passer": "^1.0.0",
```

Comment:
Did you make changes to the package-lock.json by accident? body-passer is not a real package.

Reply:
Oops ya it looks like it must have done that automatically on accident. Can someone show me how to fix that at next Casual Coding.